cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc

```diff
@@ -0,0 +1,682 @@
+
+image::vulnerability-blocked-severitiy-risk-factor.png[scale=15]
+
+//GH#26157
+|*Exceptions for Base Image Vulnerabilities*
+|For deployed and CI images, you can now https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-compute/vulnerability_management/base_images[exclude base image vulnerabilities] introduced by the base images or the middleware image while configuring the Vulnerability Management rules under *Defend > Vulnerabilities > Images > Deployed/CI*. 
+To use this feature, you need to first specify the base image under *Defend > Vulnerabilities > Images > Base images*.
+
+image::exclude-base-image-vulnerabilities.png[scale=10]
+
+When you enable this feature, the vulnerabilities that come from the base images will not be included on the https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-compute/vulnerability_management/scan_reports[scan results] view under *Monitor > Vulnerabilities > Images > Deployed/Registries/CI*.
+
+//GH#33410
+|*Alert Trigger Enhancements for Google Security Command Center*
+|The following new fields were added to existing alert triggers for Google SCC.
+
+* *Image vulnerabilities (deployed)*: Includes the following properties.
+** Collections
+** Cluster Name
+** Account ID
+
+* *Container runtime*: Includes the following properties.
+** Collections
+** Cluster Name
+** Account ID
+
+* *Incidents*: Includes the following properties.
+** Collections
+** Cluster Name
+** Account ID
+
+The container and image compliance trigger was added for Google SCC. This new trigger sends full data with every scan.
+
+//GH#34108
+|*Path and Layer Information in Syslog Output*
+|The image scan syslog output that the Prisma Cloud Console produces now includes two new fields:  `package_path` and `layer`.
+
+The host scan syslog output that the Prisma Cloud Console produces now includes one new field: `package_path`.
+
+The twistcli command line interface JSON output also shows the following new fields.
+
+* For the `images` type:
+** `package_path`
+** `layer`
+
+* For the `hosts` type:
+** `package_path`
+
+* For the `tas` type
+** `package_path`
+
+//GH#36089
+|*Regional STS Endpoint Support for Defender on AWS*
+|AWS recommends the use of a regional STS endpoint over the use of the global STS endpoint `sts.amazonaws.com`.
+When onboarding your AWS cloud account, you can now use a regional `sts.REGION.amazonaws.com` STS endpoint.
+Then, your deployed Defenders don't need to access the global STS endpoint.
+Defenders can get the STS token from the regional STS endpoint to perform scans such as registry scans.
+To enable regional STS endpoints, refer to the https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_enable-regions.html[AWS documentation].
+
+//GH#36695 - How to create a table within a row?
+
+//GH#36697
+|*Support to Generate Vulnerability Reports by Package*
```

Comment:
@divyabhushan in the title we should also mention risk factors
```suggestion
|*Support to Generate Vulnerability Reports by Package and Risk Factors*
```

Reply:
Done.